src/translations/en/common.json

```diff
@@ -3,15 +3,19 @@
   "back_to_home": "Back to home",
   "cancel": "Cancel",
   "contact_person": "Contact",
+  "create_authority": "Create author ID",
+  "description_create_authority": "Personregisteret inneholder personer med opphavsrett til publiserte vitenskapelige arbeid som er tilgjengelig i institusjonens bibliotek, vitenarkiv eller er del av Norsk Vitenskapsindeks. Også omtalte personer kan bli registrert.\n\nOm personen du søker har publisert noe vitenskapelig tidligere, eller har nylig fullført en vitenskapelig grad ved en norsk utdanningsinstitusjon, så er det sannsynlig at personen alt er registrert.\n\nEn kurator vil prøve å finne ut om dette er en ny person eller om det er en ny skrivemåte på en tidligere registrert person.",
```

Comment:
Syns ikke denne er veldig "common", men den brukes kanskje både for contributors-tab og når man skal koble til egen forfatterID ved innlogging så den passer ikke noe annet sted heller?

Reply:
enig i at det kan være litt i overkant med egen fil med bare disse to også...

Reply:
ja, jeg visste ikke helt hvor jeg skulle putte disse, siden de blir brukt på flere sider. hvis "common" skal brukes for oversettelser som finnes flere plasser, så synes jeg de passer inn her, men hvis "common" er korte generiske ord og setninger, så synes jeg ikke det passer.

Reply:
sant, blir et definisjonsspørsmål. får ligge til det evt blir et problem